### PR TITLE
fix: Mapeo no longer crashes on hybrid graphics cards

### LINKED ISSUE #279: Failed to Initialize WebGL Windows OS, Hybrid Graphics
Testing Mapeo-desktop 5.0.3 release on some windows machines results in black/blank screen. Starting the application from command line with spew out the following error in the map view. 

```
Error: Failed to initialize WebGL.

Stacktrace:

    in ReactMapboxGl (created by ForwardRef(MapViewContent))
    in ForwardRef(MapViewContent) (created by WrappedMapView)
    in WrappedMapView (created by ForwardRef(MapView))
    in ForwardRef(MapView) (created by FilterView)
    in ErrorBoundary (created by FilterView)
    in div (created by FilterView)
    in FilterView (created by MapStyleProvider)
    in MapStyleProvider (created by MapFilter)
    in div (created by MapFilter)
    in div (created by MapFilter)
    in MapFilter (created by Transition)
    in div (created by styled.div)
    in styled.div (created by Transition)
    in Transition (created by TabPanel)
    in TabPanel (created by Home)
    in div (created by styled.div)
    in styled.div (created by Home)
    in div (created by styled.div)
    in styled.div (created by Home)
    in Home (created by App)
    in IntlProvider (created by App)
    in ThemeProvider (created by App

```

http://paste.ubuntu.com/p/fS9nJkBJVR/

### PR DESCRIPTION
**Pull request by okdistribute**
### Contributor checklist:

- [x] My contribution is **not** related to translations. _Please submit translation changes via our [Mapeo Crowdin project](https://crowdin.com/project/mapeo-desktop)_
- [x] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
- [x] I have walked through the [QA Manual Testing
  Script](https://github.com/digidem/mapeo-desktop/blob/master/docs/testing.md) and updated it if necessary.
- [x] If my changes depend upon an update to a dependency, I have updated the package-lock.json file using `npm install --package-lock`
- [NA] My changes have been tested with the [mobile app](https://github.com/digidem/mapeo-mobile/releases). 
- [ ] My changes are ready to be shipped to users on Windows, Mac, and Linux

### Description

Reported by @arky that on some Windows machines with NVIDIA graphics,
Mapeo would crash on startup. This line was originally added by @noffle
but now on this version of electron doesn't seem necessary. fixes #279

### CODE REVIEW

**Comment by gmaclennan** [comment]
I'm not sure whether this will fix things.

Unfortunately, Mapeo requires WebGL to work, because it uses Mapbox for maps. WebGL is active in Chrome/Electron when it finds a compatible GPU. Some GPUs are incompatible or have bugs, so Chrome blacklists them and WebGL does not work. Adding this line was a hack to kind of force webGL to work on some linux machines even if Chrome thought they don't support WebGL, however, I think recent versions of Electron just ignore it.

From what I understand the error that @arky is seeing is because WebGL is not working on the machine, probably because the GPU is blacklisted ([there seem to be a lot of webGL problems with Nvidia drivers](https://github.com/electron/electron/issues/4380)) and because that command-line argument is already being ignored.

TL;DR - I don't think this command line argument is doing anything any more. Removing it will not make WebGL work on older machines, but there is no harm in removing it.

We should add code to detect WebGL support and display a clear message to the user. Updating drivers might be a solution for old Nvidia cards, but we'd need a test laptop to find out.

**Comment by okdistribute** [comment]
thanks @gmaclennan, I'll open an issue.
